src/main/resources/db/migration/V2__Users_relations.sql

```diff
@@ -0,0 +1,19 @@
+create table follower
+(
+    uuid             uuid primary key             not null,
+    subscriber_uuid  uuid references users (uuid) not null,
+    interest_uuid    uuid references users (uuid) not null,
+    relations_status smallint                     not null,
+    created_at       timestamp                    not null,
+    updated_at       timestamp
+);
+
+create table friend
+(
```

Comment:
Tutorial to tables
https://mysql.tutorials24x7.com/blog/guide-to-design-database-for-social-network-system-in-mysql